pkg/apis/admissionregistration/v1beta1/defaults.go

```diff
@@ -27,6 +27,30 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
+// SetDefaults_ValidatingAdmissionPolicySpec sets defaults for ValidatingAdmissionPolicySpec
```

Comment:
missing  default for a namespaceParameterRef FailAction

Reply:
Added defaulting and doc